Userland/Applications/PixelPaint/Tools/PolygonalSelectTool.cpp

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright (c) 2021, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2022, the SerenityOS developers.
```

Comment:
I don't think this file was worked on by Kling in 2021 :^)

You are free (and encouraged) to add your own copyright line

Reply:
Copy paste strikes again. I'll get this fixed on Monday

Reply:
Fixed :)